model/task/db.go

```diff
@@ -1981,8 +1982,14 @@ func GetTasksByVersion(ctx context.Context, versionID string, opts GetTasksByVer
 			} else {
 				sortFields = append(sortFields, bson.E{Key: singleSort.Key, Value: singleSort.Order})
 			}
+			if singleSort.Key == IdKey {
+				idSort = true
+			}
+		}
+		// If we're not sorting by ID, we need to sort by ID as a tiebreaker to ensure a consistent sort order.
```

Comment:
LGTM but this comment confuses me more than helps me- I don't get how it relates to the code right below

Reply:
Zack and I discussed this over Slack.
The conclusion was that it would be clearer with a better variable name.